### PR TITLE
Update notebooks to be Python 3 compatible.

### PR DESCRIPTION
**Pull request by chmeyers**
Did everything except for the MLToolbox notebooks, as those are going to take more time due to Dataflow/Cloud ML Engine dependencies.

Mostly this was just adding parens to the print statements, dealing with str/bytes issues, and changing libraries in a couple places.

### CODE REVIEW

File: tutorials/BigQuery/Importing and Exporting Data.ipynb [hunk 1]
```diff
@@ -129,7 +132,7 @@
     }
    ],
    "source": [
-    "print cars"
+    "print(str(cars, encoding='UTF-8'))"
```
Comment (yebrahim) [inline tutorials/BigQuery/Importing and Exporting Data.ipynb]: I'm surprised we need to specify the encoding, we just switched the file system encoding to UTF-8.

Comment (chmeyers) [inline tutorials/BigQuery/Importing and Exporting Data.ipynb]: For this particular case, 'cars' is a byte string and has newline characters, so if you don't change the encoding it won't print pretty, as the newlines will stay escaped.

Comment (chmeyers) [inline tutorials/BigQuery/Importing and Exporting Data.ipynb]: Just realized that this breaks in Py2, so probably just going to live with the ugly print.

Comment (chmeyers) [inline tutorials/BigQuery/Importing and Exporting Data.ipynb]: The other option would be to use 'from builtins import str' in this notebook, which would make it work in the pretty way.

**Pull request merged**